src/sentry/options/defaults.py

```diff
@@ -193,6 +193,7 @@
 register("snuba.search.max-total-chunk-time-seconds", default=30.0)
 register("snuba.search.hits-sample-size", default=100)
 register("snuba.track-outcomes-sample-rate", default=0.0)
+register("snuba.snql.referrer-rate", default=0.0)
```

Comment:
Are we still supposed to register options in a separate PR from the one we start using it to be deploy safe?
If yes I think this should go in a separate PR and land before the rest.